modules/openid_auth/app/controllers/openid_auth/v2/validation_controller.rb

```diff
@@ -81,6 +86,28 @@ def setup_structure
         end
         payload_object
       end
+
+      def charon_token_screen?(payload_object)
```

Comment:
Can this be renamed to a boolean style method (or docs added, or both) to make it clear what to expect from this?

What does it mean for a value of `true` or `false` to be returned? I had to dig into the logic to understand it.